src/renderer/components/configure-recording-set-page/index.tsx

```diff
@@ -1,268 +1,66 @@
 import log from 'electron-log';
-import React, { FC, Fragment, MouseEventHandler, useContext, useEffect, useState } from 'react';
-import Col from 'react-bootstrap/Col';
-import Container from 'react-bootstrap/Container';
-import Row from 'react-bootstrap/Row';
-import { useTranslation } from 'react-i18next';
-import CSSTransition from 'react-transition-group/CSSTransition';
-import useLocalStorage from 'react-use/lib/useLocalStorage';
+import React, { FC, MouseEventHandler, useState } from 'react';
 
-import { RecordingProjectContext } from '../../contexts';
-import { PROJECT_CONFIG_FILENAME } from '../../env-and-consts';
-import { Consumer, RecordingProject, RecordingSet, ScaleKey, SupportedOctave } from '../../types';
-import {
-  checkFileExistence,
-  deleteFolder,
-  ensureFolderExists,
-  getLSKey,
-  join,
-  naiveSerialize,
-  readFile,
-  writeFile,
-} from '../../utils';
-import BackButton from '../back-button';
-import NextButton from '../next-button';
-
-import AddRecordingSetButton from './add-recording-set-button';
-import CreatedRecordingSetList from './created-recording-set-list';
-import SetMetaConfiguration from './set-meta-configuration';
-import SetRecordingListConfiguration from './set-recording-list-configuration';
-import { BuiltInRecordingList } from './types';
+import { Consumer, RecordingSet } from '../../types';
 
+import ConfigureRecordingSet from './configure-recording-set';
 import './show-details.scss';
+import Dvcfg from './dvcfg';
```

Comment:
Actually, I was thinking, we should switch default exports to named exports because it makes things easier to refactor later on. i.e.
```
// Instead of
export default Dvcfg

// Do
export const Dvcfg = // ...
```
Can we start using this pattern with your new code?